tests/console/orphaned_packages_check.pm

```diff
@@ -38,8 +38,8 @@ sub run {
 sub post_fail_hook {
     my $self = shift;
 
-    $self->export_logs();
-    upload_logs '/tmp/orphaned.log';
+    $self->SUPER::post_fail_hook;
+    $self->upload_packagekit_logs;
```

Comment:
this change is unrelated to the "switch to tty6". Also, why use the "packagekit_logs"? Also you are loosing the log of the orphaned packages.

Reply:
I've just followed your suggestion on the ticket, what do you want to change here? any suggestion?